docs/contact_us.md

```diff
@@ -0,0 +1,46 @@
+
+
+<div class="grid cards"  markdown>
```

Comment:
i am ok to merge to get this our quickly but here are some suggestions

Put yourself first! As product manager, they should be contacting you. You can add us below (as core developers) or skip as this is a team effort and others have and will contribute to it. 

Add a big Contact Us button on the first page that goes to that page or directly to email.